Sources/Widgets/Core/StateBuilder/index.js

```diff
@@ -81,8 +81,10 @@ class Builder {
     const listName = `${name}List`;
     this.model[listName] = [];
     // Create new Instance method
-    this.publicAPI[`add${macro.capitalize(name)}`] = () => {
-      const instance = newInstance(mixins, initialValues);
+    this.publicAPI[`add${macro.capitalize(name)}`] = (values) => {
+      const allValues = { ...initialValues };
+      Object.assign(allValues, values);
+      const instance = newInstance(mixins, allValues);
```

Comment:
can you please:
 - rename values in initialValues
 - rename initialValues in defaultInitialValues
 - rename allValues in allInitialValues